desktop_version/src/preloader.cpp

```diff
@@ -88,7 +92,7 @@ void preloaderrender()
       pre_transition = 29;
     }
   }else if (pre_transition <= -10) {
-    game.gamestate=TITLEMODE;
+
```

Comment:
Might be worth just commenting this, to avoid the empty block.

Reply:
Added a comment, rebased, and force-pushed.